libraries/cbor/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,20 @@
+#![no_main]
+#[macro_use] 
+extern crate libfuzzer_sys;
+extern crate cbor;
+extern crate alloc;
+
+use alloc::vec::Vec;
+
+fuzz_target!(|data: &[u8]| {
+    let encoded = cbor::read(data);
+    match encoded{
+    	Ok(value) => {
+    		let mut decoded = Vec::new();
+    		let _ = cbor::write(value, &mut decoded);
```

Comment:
The `bool` output of `write` indicates success. It might fail if the encoded data is too deeply nested. I don't think this can happen if the structure was generated by `read` without errors. So I think you can also assert that `write` returns `true`.